package.json

```diff
@@ -51,6 +52,7 @@
     "vl-ui-action-group": "^3.1.3",
     "vl-ui-body": "^1.0.8",
     "vl-ui-button": "^5.0.10",
-    "vl-ui-util": "^5.3.10"
+    "vl-ui-info-tile": "milieuinfo/webcomponent-vl-ui-info-tile#update-dependencies",
```

Comment:
Te vervangen door nieuwe vl-info-tile release.

Reply:
Zag ik daarnet nu ook geen dependency van accordion naar info-tile? 
Circulaire dependency dan?
Zou ik proberen te vermijden nee?

Reply:
Dit is een dev dependency van vl-accordion naar vl-info-tile, te vermijden maar zorgt hier niet voor problemen.

Reply:
Das een tegenstelling :-)

Reply:
Het leven zit vol tegenstellingen. Waar onvermijdbaar is het geen probleem indien het een dev dependency is. Zelfs als het een dependency zou zijn, vormt het geen probleem dan wel aandacht bij het release.

Reply:
En ik vermoed makkelijk te vermijden door de testen dat ze samenwerken maar bij 1 van beide te steken?

Reply:
De circulaire dependency via dev is weg doordat vl-info-tile geen nood meer heeft aan de vl-accordion dependency.